projects/storefrontlib/shared/services/import-export/export.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { FileOptions } from '@spartacus/storefront';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ExportService {
+  download(fileContent: string, fileOptions: FileOptions) {
+    const { fileName, type, extension } = fileOptions;
+    const blob = new Blob(['\ufeff' + fileContent], {
+      type,
+    });
```

Comment:
This service should be generic - for creating and downloading any kind of files, right?
Therefore prepending the `\ufeff` BOM should be the responsibility of the `ExportCsvFileService#downloadCsv`